### PR TITLE
aruha-1203: changelog

### PR DESCRIPTION
**Pull request by adyach**
aruha-1203: changelog

### CODE REVIEW

**Comment by v-stepanov** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.8] - 2017-11-01
+- Massive topic deletion while switching timelines is now made with small interval between deletions
```
Comment (v-stepanov) [inline CHANGELOG.md]: There should be additional header like `### Fixed` or `### Changed`

Comment (adyach) [reply CHANGELOG.md]: done

**Comment by adyach** [comment]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.8] - 2017-11-01
+### Fixed
```
Comment (v-stepanov) [inline CHANGELOG.md]: Sorry for annoying but can you please add an empty line between version number and `Fixed` - to keep the same style for the whole file.

**Comment by adyach** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Pull request merged**